CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
-## next release
+## 2.3.0 / 2018-06-05
 
 * [CHANGE] `marathon_sd`: use `auth_token` and `auth_token_file` for token-based authentication instead of `bearer_token` and `bearer_token_file` respectively.
+* [CHANGE] Metric names for HTTP server metrics changed
+* [FEATURE] Add query commands to promtool
+* [FEATURE] Add security headers to HTTP server responses
+* [FEATURE] Pass query hints via remote read API
+* [FEATURE] Basic auth can now be configured via file across all configuration
+* [ENHANCEMENT] Optimise PromQL and API serialization for memory usage and allocations
+* [ENHANCEMENT] Limit number of dropped targets in web UI
+* [ENHANCEMENT] Add advanced filtering configuration to Consul service discovery
```

Comment:
This is still the same, and doesn't represent the change. There's no filtering that you couldn't do before. However you can now also use the filtering offered by Consul/EC2 directly for performance.

Reply:
Yes, I told you why.

Reply:
Ah, missed that.